config/default.yml

```
@@ -259,6 +259,15 @@ Gemspec/OrderedDependencies:
   Include:
     - '**/*.gemspec'
 
+Gemspec/RequireMFA:
+  Description: 'Checks that the gemspec has metadata to require MFA from Rubygems.'
```

That's just my two cents.

```suggestion
  Description: 'Checks that the gemspec has metadata to require MFA from RubyGems.'
```